lib/Doctrine/DBAL/Schema/SQLServerSchemaManager.php

```diff
@@ -37,6 +37,31 @@ class SQLServerSchemaManager extends AbstractSchemaManager
     /**
      * {@inheritdoc}
      */
+    public function dropDatabase($database)
```

Comment:
There is already a test case that covers this issue in `SchemaManagerFunctionalTestCase::testDropsDatabaseWithActiveConnections()`